front/lib/api/workspace.ts

```diff
@@ -188,12 +197,40 @@ export async function getMembersCount(
 
   return Membership.count({
     where: {
-      workspaceId: owner.id,
+      workspaceId: workspace.id,
       ...whereClause,
     },
   });
 }
 
+export async function hasWorkspaceAvailableSeats(
```

Comment:
NIT: maybe we can find better names because `hasWorkspaceAvailableSeats` and `workspaceHasAvailableSeats` are very similar? 

Or we could build an auth object on login.ts so that we don't need functions that takes a WorkspaceType | Workspace as argument? Something similar to what we have in `internalBuilderForWorkspace`? WDYT?

Reply:
Agree, I'm not a big fan. The reason I decided not to move forward with faking an auth, given that it's not for internal use but will be accessible on every login request, I'm leaning much more on the explicit side of things. 

I updated the name to reflect that one uses the Authenticator. FWIW, once we will have resource in `front` as well, we will have one unique function on the `WorkspaceResource` 🙃.